### ISSUE TITLE
Missing inflection icon for few verbs in verbpdgm34

### ISSUE THREAD

**Issue by monzug** [alpheios-project/inflection-tables#316]
such as      ἀφῆτε     subjunctive 2nd person plural in the ἵημι: Aorist System Active table

(ἄφ)ες      aorist imperative active 2nd sing plural in the ἵημι: Aorist System Active table

could it be related to the misspelled line in rules.csv?

**Comment by monzug** [comment]
it's not related to issue #315 that has been fixed. I think the lemma should be  ἀφίημι . in  rules.csv, lemma is specified for matching table verbpdgm34 as  ἵημι  only.  
also, (ἀφ)εῖμεν  should display two tables: verbpdgm34   and  verbodgm46 but only one table is displayed (and it's also 'wrong'). the stem type    irreg_mi     is ignored.

**Comment by balmas** [comment]
(1) it would probably be good to show the tables for ἵημι (e.g. verbpdgm34) for its compounds (not sure that's what they are called) such as ἀφίημι. This is similar to the issue in #291 

(2) Some forms of ἀφίημι. from table 34 produce matches on aorist system tables 6-9 because they are reported by morpheus as stemtype aor2 (e.g. ἀφείην and ἀφεῖμεν) or aor1 (e.g ἀφῆκα). Others such as ἄφες and ἀφῆτε are reported by morpheus as stemtype ath_secondary, for which we don't have any matches.

**Comment by monzug** [comment]
2) would it make any better to specify more than 1 stemtypes and more lemmas for verbpdgm34?
sing indicative is aor1
duals indicative are in Morpheus optative aor2 and irreg_mi
plural indicative ἀφεῖμεν is in morpheos optative aor2 and irreg_mi
ἀφεῖσαν 2nd person plural indicative is verb participle in Morpheos
Having some doubts on the validity of the indicative. will try to find something on smith

**Comment by balmas** [comment]
yes, I am leaning in the direction of showing more tables, rather than less. Something I want to discuss with Vanessa next week.

**Comment by monzug** [comment]
going back to 2),  ἀφεῖμεν  generates only one table, 6. Strong (2nd) Aorist System Active.
in Morpheus it's a <hdwd xml:lang="grc">ἀφίημι</hdwd>, <stemtype>aor2</stemtype>, aorist, active and <stemtype>irreg_mi</stemtype>, aorist, active
Looking at the table we should NOT generated the irreg_mi table because it's not aorist (no table verbpdgm46), but only verbpdgm34. if we fix 1) with issue #291, this one should be fixed too.
btw, line 76 in rules.csv should be deleted as it's a dup of line 75. 
I'll keep this one as a testcase to test when we have a fix for issue #291.

**Comment by monzug** [comment]
also, check that ἀφείμην , <hdwd xml:lang="grc">ἀφίημι</hdwd>, aorist, middle, 
<stemtype>ath_secondary</stemtype> has a match in table verbpdgm35 too.
